src/lua/loaders.lua

```diff
@@ -317,6 +317,22 @@ rawset(package, "search", search)
 rawset(package, "searchroot", searchroot)
 rawset(package, "setsearchroot", setsearchroot)
 
+local no_package_loaded = {}
+local raw_require = require
+
+-- Allow an override module to refuse writing to package.loaded.
+--
+-- It may be useful to return a built-in module into the platform,
+-- but the override module into application's code.
+_G.require = function(modname)
+    local mod = raw_require(modname)
```

Comment:
This line will add to any error message about `require` its location (`builtin/internal.loaders.lua:328`) -- this is why <lua-Harness-tests/241-standalone.t> is failing:
```
tarantool> string.match("LuajitError: builtin/internal.loaders.lua:328: module 'no_lib' not found:", "^[^:]+: module 'no_lib' not found:")
---
- null
tarantool> string.match("LuajitError: module 'no_lib' not found:", "^[^:]+: module 'no_lib' not found:")
---
- 'LuajitError: module ''no_lib'' not found:'
```

We can simply set `require` to the `raw_require` in <luajit-test-init.lua> to avoid this error, if we think that this line will not bother our users.

Reply:
Sorry for this mess with failed CI. I saw it yesterday but had no time to fix.

I'll spin here for a while (found other LuaJIT test fails, at least when I attempt to fix it with `pcall()` + `error(<...>, 0)` on tarantool's side).

Reply:
I changed the order of operations in the require wrapper: first drop `package.loaded`, next call `raw_require()`. This way the call is a tail call and the error messages remain the same.

Reply:
Yes, as you mentioned it changes the semantics a bit, but still OK for our usage.